### DISCUSSION TITLE
dont flag disabled automations for repair

### DISCUSSION THREAD

**Discussion by TomW1605** [frenck/spook#847] [Q&A] [answered]
Hi, just installed this and it has flaged a lot of my automations as using devices that dont exist. the thing is, they are pretty much all disabled and i keep them around for refrance (moved most of them to node red). that means i want them to keep refrancing the old, now missing, devices. it kind of makes sense to me that there should be a way to have Spook not flag these as issues. i dont want to ignore them because then a) the ignored number goes up and b) if i ever reenable that automation and forget to check it i still want it flaged.

**Comment by frenck** [comment, marked as answer]
I honestly disagree a lot. Just because automation is disabled doesn't mean it isn't loaded and processed. As a matter of fact, Home Assistant does load and process them. It is the whole reason Spook picks them up. Spook inspects runtime (not YAML files), meaning, they are there 😄 

In terms of keeping a system clean, they are just a much there as an enabled automation.

../Frenck

**Reply by TomW1605** [reply]
oh ok, i assumed they would not be processed by HA if they are disabled. in that case i might just find somewhere to archive the relavent yaml and delete the automations (or just tell spook to ignore them and put up with that number)

**Comment by atlflyer** [comment]
I agree with the OP. Being loaded and processed (whatever the latter means), isn't the same thing as being triggered and run. There are many reasons a device or entity may be unavailable on a temporary or long-term basis. Two examples might be an integration that has a breaking update or a vendor that blocks API access (temporarily, we might hope). I might want to disable the related automations instead of deleting them so when the integration is restored I have them ready to re-enable.

Could this be a configuration setting? That way we wouldn't have to come to a philosophical agreement, and instead everyone could make a practical determination about what is best for their setup. 😄
